ListingManager/Program.cs

```diff
@@ -4,10 +4,10 @@ namespace EssentialCSharp.ListingManager;
 
 public sealed class Program
 {
-    private static Task<int> Main(string[] args)
+    private static async Task<int> Main(string[] args)
```

Comment:
What is this fixing. If anything the only thing this might change is a reported stack trace at the cost of a minor bit of performance.